site/content/docs/02-template-syntax.md

```diff
@@ -887,7 +887,140 @@ Local transitions only play when the block they belong to is created or destroye
 
 ### Animations
 
-TODO i can't remember how any of this works
+```sv
+animate:name
+```
+
+```sv
+animate:name={params}
+```
+
+```js
+animation = (node: HTMLElement, { from: DOMRect, to: DOMRect } , params: any) => {
+	delay?: number,
+	duration?: number,
+	easing?: (t: number) => number,
+	css?: (t: number, u: number) => string,
+	tick?: (t: number, u: number) => void
+}
+```
+
+```js
+DOMRect {
+	bottom: number,
+	height: number,
+	​​left: number,
+	right: number,
+	​top: number,
+	width: number,
+	x: number,
+	y:number
+}
+```
+
+---
+
+An animation is triggered when the contents of a [keyed each block](docs#Each_blocks) are re-ordered. Animations do not run when an element is removed only when the each block's data is reordered. Animate directives must be on an element that is an *immediate* child of a keyed each block.
+
+Animations will normally be used with the [`flip`](docs#flip) function which will swap the positions of elements when their order changes.
```

Comment:
missing link

Reply:
It is currently missing but I felt it would be better to add it now because I'm writing the docs for `flip` as we speak. Otherwise we'd have to PR again just to add the link.